src/z3c/testing/app.py

```diff
@@ -89,20 +86,15 @@ def test_verifyClass(self):
 
     def test_verifyObject(self):
         # object test
-        self.assert_(verifyObject(self.getTestInterface(),
-            self.makeTestObject()))
+        self.assert_(verifyObject(
```

Comment:
Maybe a good opportunity to switch to a non-deprecated `self.assertTrue()` as well.